### DISCUSSION TITLE
Split spheres of specific link to two groups

### DISCUSSION THREAD

**Discussion by scharalambous-sony** [NVlabs/curobo#391] [Software Q&A] [answered]
Hi!
Does anyone know if there is a way to split the spheres of a specific link to 2 (or more) groups such that only specific groups will be ignored in self_collision_ignore?

**Comment by balakumar-s** [comment, marked as answer]
This is not possible. one option is to create additional links in the URDF that are connected to the primary link by a fixed joint.

**Reply by scharalambous-sony** [reply]
Got it, thank you
